integrations/object_store/Cargo.toml

```diff
@@ -25,7 +25,7 @@ homepage = "https://opendal.apache.org/"
 license = "Apache-2.0"
 repository = "https://github.com/apache/opendal"
 rust-version = "1.75"
-version = "0.43.1"
+version = "0.43.2"
```

Comment:
Shoud be `0.44.0` since we bumped the version of `object_store`. By the way, bumping `opendal` should also need a bump.

Reply:
OK.